packages/@cdktf/hcl2json/lib/util.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+export function wrapTerraformExpression(input: string): {
```

Comment:
(optional nit) a small doc string with an example would help a lot

Reply:
Good point! I'll do it in the next PR i'm working on. Will continue like this for now.